coinbase/coinbase-wallet-sdk
# 6492 verify message

### _Summary_

Add ability to verify eip6492 messages.  This introduces using viem.

### _How did you test your changes?_


1. yarn dev
2. Connect an scw from keys.coinbase.com
3. Click example "personal sign" , verify that message verification is successful.
<img src="https://github.com/user-attachments/assets/80afae90-8c99-475a-bc8f-1f8c9f42b95b" width="400"/>

- [x] Verify personal sign for an eoa
- [x] Verify eip 712 with scw
- [x] Verify eip712 with eoa
